### ISSUE TITLE
css-components

### ISSUE THREAD

**Issue by NikitaRzm** [Burize/FULLSTACK-Template-1#4]
Есть ли сильная мотивация иметь css-блоки кода отдельно переиспользуемыми?
Почему бы не инкапсулировать в компоненте и верстку тоже? Ведь тогда весь компонент будет легче контролировать, будет видно, каким он может быть, какие данные принимает, какие модификаторы могут его изменить и в конце концов увидеть где он используется.
Плюс появляется проблема, когда в каком-либо компоненте встречает класс типа ```point-list__row```, а на самом деле цсс для него находится принципиально в другом месте, что конечно осложняет и разработку и понимание (разумеется экстраполируя на более крупные проекты - более крупные проблемы :) ).
Я бы посоветовал собрать и .pug и . styl коды компонентов Title и point-list (другого вродне не нашел) воедино и использовать pug миксин вместо немого употребления цсс классов этих компонентов в самых неожиданных местах.

**Comment by Burize** [comment]
Да, сделано плохо (point-list был отдельно выделен; я забыл удалить старые папки в том числе папку css-components). А с Title не знал, что делать. Миксин сделать не получалось: pug не поддерживает динамический include, а иконки в заголовке были как раз таки сделаны, как компоненты. Только сегодня понял, что можно же svg, как картинку загружать, а не компонент ... Вообщем, сейчас должно быть получше.
